docs/clients/arming_handler.rst

```diff
@@ -112,42 +112,44 @@ Type ID 86 | Arming Handler
+        ||       ||                                       ||                ||          ||                         || 3 = Stow: The module will trigger a move to the stow postiion when it begins disarming. After the song, its final state will be determined by whatever the stow postiion feature is configured to do after completing a stow.                       |
+        +--------+----------------------------------------+-----------------+-----------+--------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+        || 10    || disarm_song_option                    || get, set, save || uint8    ||                         || The state that determines if and how many times the module will play its disarm song after coming to a stop. See row below for each state:                                                                                                          |
+        ||       ||                                       ||                ||          ||                         || 0 = Never Play: The disarm song is skipped entirely, and the module will transition directly to its final state after stopping.                                                                                                                     |
+        ||       ||                                       ||                ||          ||                         || 1 = Play Once: The disarm song will play once, and then the module will transition to its final state.                                                                                                                                              |
+        ||       ||                                       ||                ||          ||                         || 2 = Play Continuously: The disarm song will play continuously until the module is armed again or commanded to sping without arming through IQUART. The song will never finish, so the module will never transtion to its final state.               |
+        +--------+----------------------------------------+-----------------+-----------+--------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+        || 11    || manual_arming_throttle_source         || get, set, save || uint8    ||                         || The :ref:`throttle source <throttle_sources>` used as its armed throttle source for manual arming. The default manual throttle source is Unknown. In order to use manual arming, a throttle source must be set first. See row below for each state: |
```

Comment:
this isn't available on all firmware anymore. Probably worth mentioning that it's only on speed versions before 0.1.0